devex/cross-nuget/README.md

```diff
@@ -94,9 +94,9 @@ packaged as well as the version of the NuGet package itself:
 
 1. Change the name of the output NuGet package in all the instructions that
    follow;
-2. Change the value in `devex/cross-nuget/extras/open-enclave.nuspec`;
-3. Change `OE_SDK_TAG` in `devex/cross-nuget/linux/driver.sh`;
-4. Change `OE_SDK_TAG` in `devex/cross-nuget/windows/build.ps1`.
+2. Change the `OE_CROSS_PLAT_VERSION` in `devex/cross-nuget/extras/open-enclave.nuspec` - refer to the [NuGet page](https://www.nuget.org/packages/open-enclave-cross/) for the current version;
```

Comment:
Shouldn't this be `OE_CROSS_PLATFORM_VERSION`?

Reply:
For brevity I'll change the value to read `OE_CROSS_PLATFORM_VERSION`.

Long explanation of this value:

It's referring to the placeholder value `OE_CROSS_PLAT_VERSION` in `devex/cross-nuget/extras/open-enclave-cross.nuspec`. Since this file is essentially a template which needs to be updated and copied into a different folder during a build, for the sake of automation, it was easier using a placeholder value instead of an actual version number.